### PR TITLE
Only request replies if reply_count > 0

### PR DESCRIPTION
**Pull request by paulkaplan**
Instead of always requesting a top-level-comments replies, now that we have `reply_count` in the comment response, use that to decide whether to try loading the replies. 

For both studios and project comments.

/cc @kchadha I think we talked about this a while back, but now seems like a good time to make a bit of an optimization. 
Also /cc @picklesrus @benjiwheeler if this has any overlap with your work, but I don't think it will. 

This resolves https://github.com/LLK/scratch-www/issues/5134

### CODE REVIEW

**Review by Naleksuh — APPROVED** [review]
Looks good. Some people suggested having replies be in one request but I personally don't think that's necessary.

**Comment by paulkaplan** [comment]
Turns out there is a bug with this, which we noticed in https://github.com/LLK/scratch-www/pull/5651. I'll mark this as blocked on that one.

**Review by kchadha — APPROVED** [review]
Changes look good. We can merge this after the PR it's blocked on goes in.

**Pull request merged**
